buildtest/cli/__init__.py

```diff
@@ -1020,6 +1020,10 @@ def config_menu(subparsers, parent_parser):
     )
 
     subparsers_config.add_parser("systems", help="List all available systems")
+    #buildtest config profile remove
+    subparsers_profile_list.add_argument(
+        "-y", "--yaml", "rm", "remove", action="store_true", help="List Profile details in YAML Format"
```

Comment:
@kingxleyy this is not correct. Please refer to [add_argument](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_argument) library for reference you can't pass multiple arguments. 

Please remove `rm` and `remove`. These need to be a subparser similar to list where you can create a command `buildtest config profiles remove`.

This command should accept a positional argument that is the name of profile so if you want to delete a profile name `python` you can do

```
buildtest config profiles rm python
```

It should technically work with multiple arguments where you can remove two profiles (profile1, profile2)

```
buildtest config profiles rm profile1 profile2
```